packages/application-shell/src/components/application-entry-point/application-entry-point.tsx

```diff
@@ -8,17 +8,15 @@ import { PageUnauthorized } from '@commercetools-frontend/application-components
 import { entryPointUriPathToPermissionKeys } from '@commercetools-frontend/application-config/ssr';
 import RouteCatchAll from '../route-catch-all';
 
-type Props<AdditionalEnvironmentProperties extends {}> = {
-  environment: TProviderProps<AdditionalEnvironmentProperties>['environment'];
+type TApplicationEntryPointProps = {
+  environment: TProviderProps<{}>['environment'];
```

Comment:
This type of change is done in many places in this PR.

* Renamed the type
* Removed the type argument `AdditionalEnvironmentProperties`